src/Frontend/Core/Header/Header.php

```diff
@@ -126,7 +126,7 @@ public function __construct(KernelInterface $kernel)
         $this->meta = new MetaCollection();
 
         // add some default CSS files
-        $this->addCSS('/src/Frontend/Core/Layout/Css/screen.css');
+        $this->addCSS('/src/Frontend/Themes/' . Theme::getTheme() . '/Core/Layout/Css/screen.css');
```

Comment:
this will fail if the theme doesn't have a screen.css, with templates there is some code that checks if a file is available in a theme and that causes it to overwrite, maybe do something like that or check how it worked in 4, must have missed something